.github/workflows/generator_container_slsa3.yml

```diff
@@ -70,6 +76,14 @@ on:
         required: false
         type: string
         default: ""
+      provenance-registry-username:
```

Comment:
Using the secret as username should be discourage. Can you add a note to this option and the `registry-username`: 
"This should only be used for high entropy values such as AWS Access Key. See https://github.com/saisatishkarra/slsa-github-generator/tree/feat/cosign-repository-env/internal/builders/container#workflow-inputs" for more information.

Reply:
Added note and also readme parameters for both `provenance-registry-username` and `registry-username`